addons/l10n_in/models/chart_template.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, _
+
+
+class AccountChartTemplate(models.Model):
+    _inherit = 'account.chart.template'
+
+
+    @api.multi
+    def _prepare_all_journals(self, acc_template_ref, company, journals_dict=None):
+        res = super(AccountChartTemplate, self)._prepare_all_journals(
+            acc_template_ref, company, journals_dict=journals_dict)
+        if not self == self.env.ref('l10n_in.indian_chart_template_standard'):
+            return res
+        for journal in res:
+            if journal['code'] == 'INV':
+                journal['name'] = _('Tax Invoice')
```

Comment:
I suppose it should be 'Tax Invoices' 'Retail Invoices' 'Export Invoices'... since you'll probably have more than one, the plural  sounds more adapted